common/scripted_triggers/wc_scheme_triggers.txt

```diff
@@ -0,0 +1,29 @@
+﻿dont_excecute_scheme_trigger = {
+	scope:scheme = {
+		add_scheme_progress = -10 # Reset scheme progress
+	}
+
+	stress_impact = {
+		impatient = medium_stress_impact_gain
+		stubborn = medium_stress_impact_gain
+	}
+}
+
+abandon_scheme_trigger = {
+	scope:scheme = {
+		end_scheme = yes
+	}
+
+	stress_impact = {
+		stubborn = medium_stress_impact_gain
+	}
+}
+
+restart_scheme_trigger = {
+	play_music_cue = "mx_cue_stress"
+	custom_tooltip = restart_scheme_tt
+	
+	scope:scheme = {
+		add_scheme_progress = -10 # Reset scheme progress
+	}
+}
```

Comment:
Trigger? Check `error.log`. These are [Effects](https://ck3.paradoxwikis.com/effects), they go to scripted effects.

Reply:
Oops